src/suites/cts/copyImageBitmapToTexture.spec.ts

```diff
@@ -0,0 +1,94 @@
+export const description = `
+copy imageBitmap To texture tests.
+`;
+
+import { TestGroup } from '../../framework/index.js';
+
+import { GPUTest } from './gpu_test.js';
+
+export const g = new TestGroup(GPUTest);
+
+function calculateRowPitchForImageBitmap(width: number, bytesPerPixel: number): number {
+  const bytesPerRow = width * bytesPerPixel;
+  return (((bytesPerRow - 1) >> 8) + 1) << 8;
```

Comment:
nit: should comment what this does (rounds up to a multiple of 256)

Reply:
Done.